tests/MongoDB.Driver.Core.Tests/Core/Configuration/ConnectionSettingsTests.cs

```diff
@@ -234,10 +253,23 @@ public void With_maxLifeTime_should_return_expected_result()
             var result = subject.With(maxLifeTime: newMaxLifeTime);
 
             result.ApplicationName.Should().Be(subject.ApplicationName);
-            result.Authenticators.Should().Equal(subject.Authenticators);
+            AssertActualAuthenticatorsFactoryResult(result.AuthenticatorFactories, subject.AuthenticatorFactories);
```

Comment:
Replace with:

            result.AuthenticatorFactories.Should().Equal(subject.AuthenticatorFactories);